pandas/core/resample.py

```diff
@@ -498,23 +498,142 @@ def pad(self, limit=None):
 
     def nearest(self, limit=None):
         """
-        Fill values with nearest neighbor starting from center
+        Fill the new missing values with their nearest neighbor value, based
```

Comment:
This should fit on a single line. Can you try rephrasing?

Reply:
Lundy06032013@gmail.com

Reply:
Lundy06032013@gmail.com